<launch>
	<rosparam file="$(find map_manager)/cfg/occupancy_map_param_ugv.yaml" ns="UGV/occupancy_map"  />
    <remap from="/UGV/occupancy_map/shared_map" to= "/public_shared_map"/>
	<node pkg="map_manager" type="occupancy_map_node" name="occupancy_map_node" output="screen" ns="/UGV" />
</launch>